.pre-commit-hooks.yaml

```diff
@@ -18,3 +18,13 @@
   pass_filenames: false
   additional_dependencies: []
   minimum_pre_commit_version: "2.9.2"
+- id: uv-export
+  name: uv-export
+  description: "Automatically run 'uv export' on your project dependencies"
+  entry: uv export
+  language: python
+  files: ^uv\.lock$
+  args: ["--frozen", "--output-file=requirements.txt", "--no-emit-project"]
```

Comment:
Why `--no-emit-project` by default? cc @charliermarsh

Reply:
It feels like the more common usecase to be honest and in line with what other tools do. But that is not a hill I am willing to die on.

Reply:
I don't think we should differ from the typical defaults without strong justification, it just seems surprising.

Reply:
I'd prefer to omit it. Users can always augment the args in their own pre-commit file, right?

Reply:
Yeah definitely.